script/cibuild

```diff
@@ -6,7 +6,7 @@ set -e
 # place with this version at the beginning of each test and many commands have
 # conditional logic based on the remote version. Running the suite against
 # different major versions ensures we're covering these conditional paths.
-REMOTE_VERSIONS="2.19.0 2.21.0"
+REMOTE_VERSIONS="2.19.0 2.20.0 2.21.0"
```

Comment:
hmm, it seems we need also add 2.19 here since that is our current supported version of  ghes

https://github.com/github/backup-utils/blob/5e551d24f9fc13c40371e623ac3ac90b74ae57b5/test/bin/ghe-negotiate-version#L6

Reply:
What do you mean? 2.19 is the first one listed

Reply:
This is mostly useless these days as I don't think there are any more version-specific code paths so is really just acting as a version verifier that unnecessarily runs the entire test suite twice. Adding a third version just adds another unnecessary delay.

Reply:
hmm, I feel there are three version constraints here:
1. The absolute versions of GHES we supported: which are 2.18, 2.19, 2.20, and 2.21
2. The absolute versions of backup-util we supported: do we have it now?
3. The relative versions constraint between `backup-util` and `GHES`, which are:
Backup Utilities `major`.`minor(or feature)`.`patch` can be used to backup and restore for GHES with version from `major`.`minor-2`.`patch` to `major`.`minor`.`patch`

My question is: do we need to have version validation for ALL version constraints mentioned above?

Reply:
Ah interesting, that is a good point. Perhaps we should just drop this altogether and only run tests once...

Reply:
hmm, I feel there are three version constraints here:
1. The absolute versions of GHES we support, right now it is: 2.18, 2.19, 2.20, and 2.21
2. The absolute versions of backup-util we support: do we have it now?
3. The [relative versions constraint](https://github.com/github/backup-utils/blob/master/docs/requirements.md#github-enterprise-server-version-requirements) between `backup-util` and `GHES`, which are:
Backup Utilities `major`.`minor(or feature)`.`patch` can be used to backup and restore for GHES with version from `major`.`minor-2`.`patch` to `major`.`minor`.`patch`

My question is: do we need to have version validation for ALL version constraints mentioned above in backup-util layer?

Reply:
I don't think we _need_ to but it would certainly give us more confidence if we did!